### ISSUE TITLE
Rules:  System-based Triggers language changes

### ISSUE THREAD

**Issue by rgerrans** [openhab/openhab-docs#343]
The following needs to be modified for the behavior of 'System started' since it runs on any rule refresh, not just the ones that it is in.

```
System-based Triggers

Currently, you schedule rules to be executed either at system startup or shutdown. _Note that newly added or modified startup rules are executed once, even if openHAB is already up and running._ They are simply executed once as soon as the system is aware of them. Here’s the syntax for system triggers:

System started
System shuts down
```

Suggested edit:

Note that adding or modifying any rules will cause startup rules to execute once, even if openHAB is already up and running. 

For reference:  https://community.openhab.org/t/startup-rules-are-executed-even-though-oh-already-running-for-hours/14202?u=rgerrans

**Comment by bgilmer77** [comment]
I support the addition of the note.  This is NOT expected behavior, and could cause unintended consequences for OH2 users.  This seems like a bug, but for now, at a minimum, we should put users on notice that this behavior is normal.

**Comment by ThomDietrich** [comment]
I feel the same way. "System started" suggests a different behavior and this should (1) be documented and (2) solved.
@kaikreuzer I did a quick search and didn't find an issue where this topic is discussed. "System started" should only trigger when openHAB is started, another keyword could be introduced to execute on every change. Is there an issue or discussion we can refer to, do you feel we should start one?

**Comment by kaikreuzer** [comment]
No, there isn't any discussion.
The reason for the behaviour is rather simple: There is no "start time" of openHAB, as all components can come and go dynamically. So the system makes sure that every "system started" rule is executed as early as possible as soon as the system knows about it.

I wouldn't want to technically change anything there until we might have https://github.com/eclipse/smarthome/issues/1896 in place, which would give us a proper "system started event".

**Comment by bgilmer77** [comment]
Thank you Kai for the explanation.  I have been running OH1 for a while and am now moving to OH2.  As a *NIX SysAdmin and native English speaker, I want to give back by contributing to the documentation, and I have just started working with Thom to learn how to do that.  I will contribute a little text around this to help others who come after me understand the behavior of the System started trigger.

**Comment by kaikreuzer** [comment]
Many thanks @bgilmer77, I very much appreciate your efforts!

**Comment by bgilmer77** [comment]
@kaikreuzer I have created a pull request, adding a little text to explain the behavior of the System started trigger at #416 .  You might want to look at it to see if I have described the behavior correctly.  This is the behavior I experience on my system.
Thanks.

**Comment by LightIsLife** [comment]
@ThomDietrich is still pertinent? Seems it got resolved in #416

**Comment by ThomDietrich** [comment]
It did! Thanks ;)
